packages/protocol/lib/mento-core

```diff
@@ -0,0 +1 @@
+Subproject commit 7c36d6031e0d1ed04582a38dd5de0c50f04d6cfc
```

Comment:
Should we consider using their master? In such case if there would be any breaking change we would know immediately.

Reply:
The idea would be to use what they currently have in production. Changes are supposed to be backward compatible, if not I guess we should be notified accordingly.

Reply:
In such case should we have some process to update the commit when Mento will make new release ?

Reply:
I'm guessing that Mento release and our release have different cadence so it would have to updated in Mento release